crates/autopilot/src/limit_orders/metrics.rs

```diff
@@ -0,0 +1,43 @@
+use chrono::Duration;
+use prometheus::IntGauge;
+
+use crate::database::Postgres;
+
+pub struct LimitOrderMetrics {
+    pub limit_order_age: Duration,
+    pub loop_delay: std::time::Duration,
+    pub database: Postgres,
+}
+
+impl LimitOrderMetrics {
+    pub fn spawn(self) {
+        tokio::spawn(async move {
+            let limit_orders_gauge = gauge("limit_orders", "Valid limit orders.");
+            let quoted_limit_orders_gauge = gauge("quoted_limit_orders", "Quoted limit orders.");
+            let unquoted_limit_orders_gauge = gauge(
+                "unquoted_limit_orders",
+                "Unquoted or outdated limit orders.",
+            );
+            loop {
+                let limit_orders = self.database.count_limit_orders().await.unwrap();
+                let unquoted_limit_orders = self
+                    .database
+                    .count_limit_orders_with_outdated_fees(self.limit_order_age)
+                    .await
+                    .unwrap();
+                let quoted_limit_orders = limit_orders - unquoted_limit_orders;
```

Comment:
If we're pretty sure that this always how this is going to be calculated then no need to make this a separate counter. Can do the same operation in prometheus/grafana. That saves a little bit of space in prometheus. On the other hand maybe this is cleaner from another point of view. Not sure.